tests/parser/functions/test_block_number.py

```diff
@@ -1,6 +1,6 @@
 
-def test_block_number(get_contract_with_gas_estimation, chain):
-    chain.mine(1)
+def test_block_number(get_contract_with_gas_estimation, tester):
+    tester.mine_blocks(1)
```

Comment:
Just in case you aren't aware, `w3.testing.mine(n)` can be used if you want to stick with web3 APIs

Reply:
neat, yes I think having tests use `web3` as much as possible is the way to go.